lib/rt_rubocop_defaults/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module RTRuboCopDefaults
-  VERSION = "2.1.0"
+  VERSION = "2.2.0"
```

Comment:
Do the changes require a major version upgrade?

Reply:
no, minor version bump is enough